system/Debug/Toolbar.php

```diff
@@ -468,4 +468,27 @@ protected function format(string $data, string $format = 'html'): string
 
 		return $output;
 	}
+
+	/**
+	 * Process the varable to string for display
+	 *
+	 * @param mixed $var Variable
+	 *
+	 * @return boolean|string
+	 */
+	protected function processVar($var)
+	{
+		if (is_object($var))
+		{
+			$var = is_callable([$var, 'toArray']) ? $var->toArray() : get_class($var);
```

Comment:
This no longer a) tells us the object class for array-ables nor b) allows displaying valid, non-recursive objects that are not array-able.